ui/src/app/components/inputs.tsx

```diff
@@ -82,6 +83,132 @@ export const ValidationError = ({children}) => {
   >{children}</div>;
 };
 
+export const TextArea = ({style = {}, onChange, ...props}) => {
+  return <textarea
+      {...props}
+      onChange={onChange ? (e => onChange(e.target.value)) : undefined}
+      style={{
+        width: '100%',
+        borderColor: inputBorderColor, borderWidth: 1, borderStyle: 'solid', borderRadius: 3,
```

Comment:
nit: Extract Style

Reply:
I deliberately didn't extract the style because it's not reused and it's in a file that has multiple different classes.